src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonUtilities.Defaults.cs

```diff
@@ -31,7 +34,7 @@ private static JsonSerializerOptions CreateDefaultOptions()
             JsonSerializerOptions options = new(JsonSerializerDefaults.Web)
             {
                 TypeInfoResolver = new DefaultJsonTypeInfoResolver(),
-                Converters = { new JsonStringEnumConverter() },
+                Converters = { new JsonStringEnumConverter(), new JsonStringBooleanConverter() },
```

Comment:
I should point out that this configuration is going to be lost as soon as users stop using the default JSO. This is a fairly common scenario, particularly when using AOT and source generators.

The only real workaround is exposing the custom converter so that users remember to plug it into their own configuration, however adding STJ extension APIs to M.E.AI isn't something we are very keen on doing unless it represents a substantial blocker.